### PR TITLE
AAP-19133 Update content on teams for 2.5 RBAC changes

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR includes RBAC changes for 2.5 for content on Teams.

### CODE REVIEW

File: downstream/modules/platform/proc-controller-creating-a-team.adoc [hunk 1]
```diff
@@ -1,50 +1,23 @@
...
-* To remove roles for a particular user, click the image:disassociate.png[Disassociate,10,10] icon next to its resource.
-
-//image:permissions-disassociate.png[image]
-
-This launches a confirmation dialog, asking you to confirm the disassociation.
-
-//image:permissions-disassociate-confirm.png[image]
+. Click btn:[Create team].
++
+The Details page opens, where you can review and edit your team information.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-creating-a-team.adoc:22]: The * Details * page

File: downstream/modules/platform/proc-controller-user-permissions.adoc [hunk 2]
```diff
@@ -1,38 +1,22 @@
...
-This feature reduces the time for an organization to onboard many users at one time.
-====
-
-.To remove permissions:
-* Click the image:disassociate.png[Disassociate,10,10] icon next to the resource. 
-This launches a confirmation dialog asking you to confirm the disassociation.
-
-
+. Select a *Resource type* and click btn:[Next]. 
+. Select the resources that will receive new roles and click btn:[Next].
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-user-permissions.adoc:18]: Select the resource s to receive the new roles...

File: downstream/modules/platform/proc-controller-user-permissions.adoc [hunk 3]
```diff
@@ -1,38 +1,22 @@
...
+. Select the roles that will be applied to the resources and click btn:[Next].
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-user-permissions.adoc:19]: Select the roles to apply to the resources...

**Review by ianf77 — COMMENTED** [review]
Couple of comments

File: downstream/modules/platform/proc-controller-user-permissions.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-user-permissions.adoc:18]: Fixed.

File: downstream/modules/platform/proc-controller-user-permissions.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-user-permissions.adoc:19]: Fixed.

**Review by ianf77 — COMMENTED** [review]
LGTM

**Pull request merged**